src/components/select/editForm/Select.edit.validation.js

```diff
@@ -2,12 +2,32 @@ export default [
   {
     weight: 50,
     type: 'checkbox',
-    label: 'Perform server validation',
+    label: 'Perform server validation of remote value',
     tooltip: 'Check this if you would like for the server to perform a validation check to ensure the selected value is an available option. This requires a Search query to ensure a record is found.',
     key: 'validate.select',
     input: true,
     conditional: {
       json: { var: 'data.searchField' }
     }
+  },
+  {
+    weight: 52,
+    type: 'checkbox',
+    label: 'Perform server validation of static value',
```

Comment:
I am really confused why we need to make a distinction of ```server validation```.  Why can we just not make this a setting that is ```Perform validation of select values``` and is executed on both client and server when checked. Otherwise, it will just skip this validation?  I cannot think of a use case where someone would want to validate it on the server, but not also on the client. They either want it all on or all off. Having these two separate configurations is confusing.